guides/common/modules/proc_configuring-foreman-repositories-el-8.adoc

```diff
@@ -0,0 +1,34 @@
+[id="configuring-foreman-repositories-el-8"]
```

Comment:
This looks like an exact copy of el7 with `s/el7/el8/g`. Can that somehow be combined with something like a major version variable?

Reply:
Consolidated into a single guide using a variable.